app/models/comparison/annual_electricity_costs_per_pupil.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: annual_electricity_costs_per_pupils
+#
+#  alert_generation_run_id                    :bigint(8)
+#  id                                         :bigint(8)
+#  last_year_gbp                              :float
+#  one_year_electricity_per_pupil_gbp         :float
+#  one_year_saving_versus_exemplar_gbpcurrent :float
+#  school_id                                  :bigint(8)
+#
+class Comparison::AnnualElectricityCostsPerPupil < Comparison::View
+  scope :with_data, -> { where.not(one_year_electricity_per_pupil_gbp: nil) }
```

Comment:
The original doesn't have this condition but it seems like a good idea to make sure there is some data before displaying the school?

Reply:
Yes, seems sensible